### PR TITLE
Implement PM2 for process management - Closes #98

### LINKED ISSUE #98: Replace Forever with PM2
To maintain consistency between lisk-core and lisk-build. Forever should be replaced with PM2

Readme.md

reference: https://github.com/LiskHQ/lisk-build/issues/91

### PR DESCRIPTION
**Pull request by Isabello**
Forever is not maintained so well. And in order to stay in sync with Core, we are switching to PM2 to manage explorer processes. This PR adds the config file, and necessary folders for PM2 to operate. It also includes new instructions for using pm2.

Closes #98

### CODE REVIEW

File: pm2-explorer.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+        "apps": [{
+                "name": "explorer.app",
```
Comment (4miners) [inline pm2-explorer.json]: `lisk-explorer` is better name ;)

File: pm2-explorer.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+                "script": "app.js",
+                "cwd": ".",
+                "pid_file": "./pids/explorer.app.pid",
+                "out_file": "./logs/explorer.app.log",
+                "error_file": "./logs/lisk.app.err",
+                "log_date_format": "YYYY-MM-DD HH:mm:ss SSS",
+                "watch": false,
+                "kill_timeout" : 10000,
+                "max_memory_restart": "512M",
```
Comment (4miners) [inline pm2-explorer.json]: Not sure if 512 will be enough, but worth a try.

File: pm2-explorer.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+        "apps": [{
+                "name": "explorer.app",
+                "script": "app.js",
+                "cwd": ".",
+                "pid_file": "./pids/explorer.app.pid",
+                "out_file": "./logs/explorer.app.log",
```
Comment (4miners) [inline pm2-explorer.json]: Would be nice to use something like `pm2-logrotate` and limit log size, ppl have issues with logs fill entire disk space over and over again.

File: pm2-explorer.json (hunk 2 above)
Comment (Isabello) [inline pm2-explorer.json]: Could go higher. Can you check your nodes and give me some more data?

File: pm2-explorer.json (hunk 3 above)
Comment (Isabello) [inline pm2-explorer.json]: I will do.

File: pm2-explorer.json (hunk 1 above)
Comment (Isabello) [inline pm2-explorer.json]: Yes ok. I debated but was unsure.

File: pm2-explorer.json (hunk 2 above)
Comment (4miners) [inline pm2-explorer.json]: It's 192MB currently - so 512 should be enough.

Comment (Isabello) [inline pm2-explorer.json]: thanks!

File: pm2-explorer.json (hunk 3 above)
Comment (Isabello) [inline pm2-explorer.json]: this is a seperate module, not something we implement in the config file. I will add a step to the readme to indicate that it should be installed.

**Comment by Isabello** [comment]
@4miners

File: pm2-explorer.json (hunk 3 above)
Comment (4miners) [inline pm2-explorer.json]: Yep, great. :)

File: README.md [hunk 4]
```diff
@@ -37,9 +37,10 @@ These programs and resources are required to install and run Lisk Explorer
 
   `sudo npm install -g grunt`
 
-- Forever (<https://github.com/foreverjs/forever>) -- Forever manages the node processes for Lisk Explorer
+- PM2 (https://github.com/Unitech/pm2) -- PM2 manages the node process for Lisk Explorer and handles log rotation (Highly Recommended)
 
-  `sudo npm install -g forever`
+  `sudo npm install -g pm2`
+  `pm2 install pm2-logrotate`
```
Comment (4miners) [inline README.md]: Would be nice to describe how to configure `pm2-logrotate` correctly (`max_size`, `compress`, etc.)

**Comment by Isabello** [comment]
Ill give a link and suggested defaults

**Pull request merged**